src/pages/NotificationUserConfigPage/hooks/useNotificationConfigHook.ts

```diff
@@ -0,0 +1,256 @@
+                'delegation.delegationAssignment.components.delegationUpdated.description'
+              ),
+              visibility: ['admin'],
+            },
+            {
+              key: 'eserviceNewVersionSubmittedToDelegator',
+              title: t('delegation.delegationAssignment.components.eserviceDelegatedCreated.label'), // 21
+
+              description: t(
+                'delegation.delegationAssignment.components.eserviceDelegatedCreated.description'
+              ),
+              visibility: ['admin'],
+            },
+          ],
+        },
+        {
+          name: 'delegationReceive',
+          title: t('delegation.delegationReceive.title'),
+          components: [
+            {
+              key: 'delegationSubmittedRevokedToDelegate',
+              title: t('delegation.delegationReceive.components.delegationUpdated.label'), // 23
+              description: t(
+                'delegation.delegationReceive.components.delegationUpdated.description'
+              ),
+              visibility: ['admin'],
+            },
+            {
+              key: 'eserviceNewVersionApprovedRejectedToDelegate',
+              title: t('delegation.delegationReceive.components.eserviceDelegatedApproval.label'), // 22
+              description: t(
+                'delegation.delegationReceive.components.eserviceDelegatedApproval.description'
+              ),
+              visibility: ['admin', 'api'],
+            },
+          ],
+        },
+      ],
+    },
+    keyAndAttributes: {
+      title: t('keyAndAttributes.title'),
+      icon: CodeIcon,
+      subsections: [
+        {
+          name: 'attributes',
+          title: t('keyAndAttributes.attributes.title'),
+          components: [
+            {
+              key: 'certifiedVerifiedAttributeAssignedRevokedToAssignee',
+              title: t('keyAndAttributes.attributes.components.attributesStateUpdated.label'), // 24
+              description: t(
+                'keyAndAttributes.attributes.components.attributesStateUpdated.description'
+              ),
+              visibility: ['admin'],
+            },
+          ],
+        },
+        {
+          name: 'keys',
+          title: t('keyAndAttributes.keys.title'),
+          components: [
+            {
+              key: 'clientKeyAndProducerKeychainKeyAddedDeletedToClientUsers',
+              title: t('keyAndAttributes.keys.components.clientKeysAssociationUpdated.label'), // 25
+              description: t(
+                'keyAndAttributes.keys.components.clientKeysAssociationUpdated.description'
+              ),
+              visibility: ['admin', 'security'], // To define
+            },
+          ],
+        },
+      ],
+    },
+  }
+
+  const notificationSchema: NotificationConfigSchema = match(type)
+    .with('inApp', () => {
+      return notificationConfigSchema
+    })
+    .with('email', () => {
+      return notificationConfigSchema
+    })
+    .exhaustive()
```

Comment:
Is this for a future implementation when the `inApp` and `email` have their own scema?